### PR TITLE
bfgminer 5.5.0 (new formula)

### PR DESCRIPTION
**Pull request by Ishaanahuja7**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/bfgminer.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  on_linux do
+    depends_on "curl"
+  end
+
+  def install
+    libgcrypt = Formula["libgcrypt"]
+    libusb = Formula["libusb"]
+
+    ENV.append "CPPFLAGS", "-I#{libgcrypt.opt_include} -I#{libusb.opt_include}/libusb-1.0"
+    ENV.append "LDFLAGS", "-L#{libgcrypt.opt_lib} -L#{libusb.opt_lib}"
```
Comment (SMillerDev) [inline Formula/bfgminer.rb:30]: Is it no able to find these using `pkg-config` or something?

File: Formula/bfgminer.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "uthash" => :build
+  depends_on "jansson"
+  depends_on "libevent"
+  depends_on "libusb"
+
+  on_linux do
+    depends_on "curl"
+  end
```
Comment (SMillerDev) [inline Formula/bfgminer.rb:23]:
```suggestion
  uses_from_macos "curl"
```

File: Formula/bfgminer.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
+class Bfgminer < Formula
+  desc "Modular CPU/GPU/ASIC/FPGA miner written in C"
+  homepage "http://bfgminer.org"
+  url "http://bfgminer.org/files/latest/bfgminer-5.5.0.txz"
+  sha256 "ac254da9a40db375cb25cacdd2f84f95ffd7f442e31d2b9a7f357a48d32cc681"
+  license "GPL-3.0-or-later"
+  head "https://github.com/luke-jr/bfgminer.git", branch: "bfgminer"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
```
Comment (SMillerDev) [inline Formula/bfgminer.rb:10]: Are these needed? it doesn't seem to reconfigure anything.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @Ishaanahuja7!

**Pull request closed without merging**